modules/fbs/fbs.info

```diff
@@ -9,13 +9,15 @@ dependencies[] = ding_provider
 dependencies[] = ding_reservation
 dependencies[] = ding_user
 dependencies[] = entity
+dependencies[] = fbs
```

Comment:
This will make `fbs` depend on itself. Features might be silly but that does not seem right?